### ISSUE TITLE
Spec should not be hosted under https://identity.foundation/.well-known

### ISSUE THREAD

**Issue by dwaite** [decentralized-identity/well-known-did-configuration#74]
Per the RFC, data within /.well-known/ is not meant to be a new general namespace, but rather used for service discovery. Hosting non-metadata information such as the specification within /.well-known/ is outside the spirit of the RFC - which is a bit odd for a specification trying to define usage of well-known for a particular purpose.

Generally, if you aren't willing to publish an RFC about a resource under /.well-known/ and have it within the [IANA registry](https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml), it does not belong under the /.well-known/ path.

**Comment by tmarkovski** [comment]
I agree with you. I also don't think this was done intentionally, just a side effect of the repository name and how GitHub pages work with published content 😀

**Comment by jceb** [comment]
We move the spec to a new repository and new subpath.
